internal/cmd/config/config.go

```diff
@@ -95,10 +97,12 @@ type Config struct {
 }
 
 type Controller struct {
-	Name              string    `hcl:"name"`
-	Description       string    `hcl:"description"`
-	Database          *Database `hcl:"database"`
-	PublicClusterAddr string    `hcl:"public_cluster_addr"`
+	Name                  string    `hcl:"name"`
+	Description           string    `hcl:"description"`
+	Database              *Database `hcl:"database"`
+	PublicClusterAddr     string    `hcl:"public_cluster_addr"`
+	AuthTokenMaxDuration  string    `hcl:"auth_token_max_duration"`
+	AuthTokenMaxStaleness string    `hcl:"auth_token_max_staleness"`
```

Comment:
This isn't really a "max" staleness value -- or if it is, I don't think that's something that's been discussed, because it implies overriding from auth methods isn't ever possible. That also applies to the MaxDuration above.

Likely these need to be something like `default_auth_token_ttl` and `default_auth_token_staleness_duration` or something like that.

Reply:
Yeah the wording here was a little tricky. I based the current implementation off the existing wording for the vars we formerly used in the repo code, but I'm happy to swap these out for something more agreeable. 
 
Do we want to use the word `default` in the attribute name? I was thinking the setting would _override the default_ we set in the code. I like:
- `auth_token_ttl`
- `auth_token_staleness_duration`